Makefile

```diff
@@ -3,11 +3,12 @@ BUILD_FILES = $(shell go list -f '{{range .GoFiles}}{{$$.Dir}}/{{.}}\
 
 GH_VERSION ?= $(shell git describe --tags 2>/dev/null || git rev-parse --short HEAD)
 DATE_FMT = +%Y-%m-%d
-ifdef SOURCE_DATE_EPOCH
-    BUILD_DATE ?= $(shell date -u -d "@$(SOURCE_DATE_EPOCH)" "$(DATE_FMT)" 2>/dev/null || date -u -r "$(SOURCE_DATE_EPOCH)" "$(DATE_FMT)" 2>/dev/null || date -u "$(DATE_FMT)")
-else
-    BUILD_DATE ?= $(shell date "$(DATE_FMT)")
-endif
+BUILD_DATE ?= $(shell \
+	2>/dev/null \
```

Comment:
It looks like `2>/dev/null` appears before the `date` command, but I would imagine it appearing after. Is all the same, and did you intend for the redirect to be at the start?

Reply:
Yes, I intended it to be at the start, it has the same effect.